apps/condo/domains/meter/constants/constants.js

```diff
@@ -11,6 +11,7 @@ const MESSENGER_METER_READING_SOURCE_ID = 'e5f3d3d3-abc2-48a8-99e8-33ade60ea205'
 const SOCIAL_NETWORK_METER_READING_SOURCE_ID = 'e07c307f-1c7a-4da6-87b9-1c03a990a19d'
 const MOBILE_APP_STAFF_METER_READING_SOURCE_ID = '43c9fd8e-802f-4e8a-98b9-532596d7f8c1'
 const CRM_METER_READING_SOURCE_ID = 'b09fec04-5bc2-4c58-a364-04b1ea505503'
+const METERS_BILLING_INTEGRATION_KVP24_READING_SOURCE_ID = '3450106c-2613-49e8-8d97-b3adf5edd189'
```

Comment:
I do not really like the idea of moving some internal kvp24 stuff (it is a b2b app afaik) to open source condo part

Reply:
You right& Renamed to "external integration"